src/sandstorm/identity.capnp

```diff
@@ -39,6 +39,13 @@ interface Identity {
 
   # TODO(someday): Public key info? Ability to seal messages / check signatures?
 
+  struct PowerboxTag {
+    # Tag to be used in a `PowerboxDescriptor` to describe an `Identity`.
+
+    identityId @0 :Data;
```

Comment:
OK, I've removed `PowerboxTag.identityId` and added `SandstormApi.getIdentityId(identity)`, which is the option that I think has the easiest-to-understand transition path.

It occurs to me that `PowerboxTag.permissions` might eventually require us to do some kind of descriptor interception, at least on the fulfillment side. However, I'm not to worried about that, as it seems like it would be relatively straightforward to handle.